src/main/webapp/app/resources/css/style.css

```diff
@@ -0,0 +1,29 @@
+@charset "ISO-8859-1";
```

Comment:
/resources folder should stay outside webapp folder.

Reply:
done

Reply:
still si there : ![image](https://user-images.githubusercontent.com/5796627/109401226-bb7e6e80-794d-11eb-9fc9-bc4e4e0e1789.png)